addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -460,7 +460,7 @@ def _import_retrieve_partner_vals(self, tree, role):
     def _export_invoice(self, invoice, convert_fixed_taxes=True):
         # Only for BIS3 invoices: if the 'account_edi_ubl_cii.use_new_dict_to_xml_helpers' param is set,
         # use the new dict_to_xml helpers.
-        if self._name == 'account.edi.xml.ubl_bis3' and self.env['ir.config_parameter'].sudo().get_param('account_edi_ubl_cii.use_new_dict_to_xml_helpers'):
+        if self._name == 'account.edi.xml.ubl_bis3' and self.env['ir.config_parameter'].sudo().get_param('account_edi_ubl_cii.use_new_dict_to_xml_helpers') not in {'0', 'f', 'false', 'False'}:
```

Comment:
```suggestion
        if self._name == 'account.edi.xml.ubl_bis3' and str2bool(self.env['ir.config_parameter'].sudo().get_param('account_edi_ubl_cii.use_new_dict_to_xml_helpers')):
```
(from `from odoo.tools.misc import str2bool`)